docs/specs/git.yml

```diff
@@ -101,6 +100,75 @@ outputs:
   build.manifest:
   dep.json:
 ---
+# Contributors can be excluded, it's case insensitive
+repo: https://github.com/docascode/docfx-test-dependencies#git-test-4
```

Comment:
it is using `repo` or `inputs` or `both`?

Reply:
both

---
In reply to: [210815294](https://github.com/dotnet/docfx/pull/3224#discussion_r210815294) [](ancestors = 210815294)